archivist_samples/testing/asset.py

```diff
@@ -0,0 +1,215 @@
+#   Copyright 2019 Jitsuin, inc
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# WARNING: Proof of concept code: Not for release
+# Copies the flow of the Jitsuinator demo script
+
+"""Class that implements common actions on an asset. This is the secret sauce
+   and this code may be added to the python SDK at some later date...
+"""
+
+# pylint:  disable=missing-docstring
+
+from archivist.timestamp import make_timestamp
+
+from .namespace import (
+    events_create,
+)
+
+CONFIG_MANAGEMENT = "Config Management"
+FW_UPDATE = "FW Update"
+FW_VULNERABILITY = "FW Vulnerability"
+OPERATIONAL_REPORT = "Operational Report"
+SERVICE_RP = "Service RP"
+SERVICE_RQ = "Service RQ"
```

Comment:
given how they're used above these should match the pairs we agreed for the SaaS service updates: 
* 'Vulnerability Report'
* 'Vulnerability Addressed'
* 'Maintenance Request'
* 'Maintenance Performed'

Reply:
fixed